docs/azure_jumpstart_ag/contoso_supermarket/freezer_monitor/_index.md

```diff
@@ -4,4 +4,261 @@ weight: 100
+
+- In the Grafana dashboard, click the hamburger menu next to __Home__ then __Alerting__
+
+  ![Grafana showing the Alerting menu](./img/grafana_click_alerting.png)
+- Add a Contact point
+  - Click __Contact points__ in the navigation menu on the left
+
+    ![Grafana showing the Contact points menu](./img/grafana_click_contact_points.png)
+  - Click the __Add contact point__ button
+
+    ![Grafana showing the Add contact point button](./img/grafana_click_add_contact_point.png)
+  - Enter _Chicago Store Manager_ as the __Name__ and _chicago@contoso.com_ in the __Addresses__ field then click __Save contact point__
+
+    ![Grafana showing the Add contact point form](./img/grafana_add_contact_point.png)
+- Add an Alert rule
+  - Click __Alert rules__ in the navigation menu
+
+    ![Grafana showing the Alert rules menu](./img/grafana_click_alert_rules.png)
+  - Click __Create alert rule__
+
+    ![Grafana showing the Create alert rule button](./img/grafana_click_create_alert_rule.png)
+  - Section 1
+    - Enter _Freezer too warm - food at risk_ as the __Rule name__
+
+      ![Grafana showing the Add alert rule form](./img/grafana_add_alert_rule_name.png)
+  - Section 2
+    - Select __chicago__ as the data source in query __A__
+    - Select __temperature__ as the Metric in query __A__
+    - Enter _15_ as the Threshold in expression __C__
+    - Click __Preview__
+
+      ![Grafana showing the Add alert rule form](./img/grafana_add_alert_rule_config.png)
+    - View the preview
+
+      ![Grafana showing the Add alert preview](./img/grafana_add_alert_rule_preview.png)
+    - Since it's difficult to determine which series is which, let's fix the series names
+      - Click __Options__
+      - Under __Legend__ select __Custom__ then enter _{{sensor_type}}_
+      - Click away from the 'Legend' box, then back into it for the series names to update
+
+        ![Grafana showing preview legend options](./img/grafana_add_alert_rule_preview_legend.png)
+    - Notice in the chart the times where freezer2 exceeds the threshold and would trigger an alert
+  - Section 3
+    - Under __Folder__ type _Alerts_ and press __Enter__
+    - Under __Evaluation group__ type _Alert Group_ and press __Enter__
+
+      ![Grafana showing the Folder selection](./img/grafana_add_alert_rule_folder.png)
+  - Scroll to the top of the page and click __Save and exit__
+
+    ![Grafana showing the Save and exit button](./img/grafana_add_alert_rule_save.png)
+- View your alert
+  - Back on the __Alert rules__ page, type _freezer_ in the __Search__ box and press __Enter__
+  - Expand the __Alerts > Alert Group__ folder to see your __Freezer too warm - food at risk__ alert
+  - Expand the alert to view to __Silence__ it, __Show state history__, or quickly __edit__ or __delete__ the alert
+
```

Comment:
this entire section is not using blank lines between bullets. Is there a formatting reason why or can we make it consistent with other bullets in the doc?

Reply:
fixed